diff --git a/dist/private/color/Color.cjs b/dist/private/color/Color.cjs
index e4b73ccb34e60746e834cfdb73245b92dfa81c20..a55f4c43c5cd37757f4c2e56e5865622887a5aa4 100644
--- a/dist/private/color/Color.cjs
+++ b/dist/private/color/Color.cjs	
@@ -387,13 +387,11 @@ class $90c404755d945e7c$var$RGBColor extends $90c404755d945e7c$var$Color {
     getColorSpace() {
         return 'rgb';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'red',
             'green',
             'blue'
         ];
-    }
     getColorChannels() {
         return $90c404755d945e7c$var$RGBColor.colorChannels;
     }
@@ -527,13 +525,11 @@ class $90c404755d945e7c$var$HSBColor extends $90c404755d945e7c$var$Color {
     getColorSpace() {
         return 'hsb';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'hue',
             'saturation',
             'brightness'
         ];
-    }
     getColorChannels() {
         return $90c404755d945e7c$var$HSBColor.colorChannels;
     }
@@ -667,13 +663,11 @@ class $90c404755d945e7c$var$HSLColor extends $90c404755d945e7c$var$Color {
     getColorSpace() {
         return 'hsl';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'hue',
             'saturation',
             'lightness'
         ];
-    }
     getColorChannels() {
         return $90c404755d945e7c$var$HSLColor.colorChannels;
     }
diff --git a/dist/private/color/Color.mjs b/dist/private/color/Color.mjs
index da24a9fc021027d1ceb4e2d15e8c4bc8d6b92de5..150e69e8ecc00ca4052f5741e07d3e10fcbe5680 100644
--- a/dist/private/color/Color.mjs
+++ b/dist/private/color/Color.mjs	
@@ -379,13 +379,11 @@ class $890783418b00a858$var$RGBColor extends $890783418b00a858$var$Color {
     getColorSpace() {
         return 'rgb';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'red',
             'green',
             'blue'
         ];
-    }
     getColorChannels() {
         return $890783418b00a858$var$RGBColor.colorChannels;
     }
@@ -519,13 +517,11 @@ class $890783418b00a858$var$HSBColor extends $890783418b00a858$var$Color {
     getColorSpace() {
         return 'hsb';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'hue',
             'saturation',
             'brightness'
         ];
-    }
     getColorChannels() {
         return $890783418b00a858$var$HSBColor.colorChannels;
     }
@@ -659,13 +655,11 @@ class $890783418b00a858$var$HSLColor extends $890783418b00a858$var$Color {
     getColorSpace() {
         return 'hsl';
     }
-    static{
-        this.colorChannels = [
+    static colorChannels = [
             'hue',
             'saturation',
             'lightness'
         ];
-    }
     getColorChannels() {
         return $890783418b00a858$var$HSLColor.colorChannels;
     }
