diff --git a/dist/private/Breadcrumbs.cjs b/dist/private/Breadcrumbs.cjs
index 75d5c8330ab9623d3745b68288f6fb60b22e7b76..74658fa8852ed8cbf007abffdec4a784afbdf86e 100644
--- a/dist/private/Breadcrumbs.cjs
+++ b/dist/private/Breadcrumbs.cjs	
@@ -66,9 +66,7 @@ const $c3765ed6efe4e06d$export$2dc68d50d56fbbd = /*#__PURE__*/ (0, $lMF0i$react.
         }))));
 });
 class $c3765ed6efe4e06d$var$BreadcrumbNode extends (0, $lMF0i$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $c3765ed6efe4e06d$export$dabcc1ec9dd9d1cc = /*#__PURE__*/ (0, $lMF0i$reactariaCollectionBuilder.createLeafComponent)($c3765ed6efe4e06d$var$BreadcrumbNode, function Breadcrumb(props, ref, node) {
     // Recreating useBreadcrumbItem because we want to use composition instead of having the link builtin.
diff --git a/dist/private/Breadcrumbs.mjs b/dist/private/Breadcrumbs.mjs
index 872c8f23ba0f9d476e56fde378c4dd591d673f10..b6eda5b7dc868c08f4b5aaf6c44f06d0239ea98a 100644
--- a/dist/private/Breadcrumbs.mjs
+++ b/dist/private/Breadcrumbs.mjs	
@@ -54,9 +54,7 @@ const $65dbe90f868fa5f4$export$2dc68d50d56fbbd = /*#__PURE__*/ (0, $hEiKI$forwar
         }))));
 });
 class $65dbe90f868fa5f4$var$BreadcrumbNode extends (0, $hEiKI$CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $65dbe90f868fa5f4$export$dabcc1ec9dd9d1cc = /*#__PURE__*/ (0, $hEiKI$createLeafComponent)($65dbe90f868fa5f4$var$BreadcrumbNode, function Breadcrumb(props, ref, node) {
     // Recreating useBreadcrumbItem because we want to use composition instead of having the link builtin.
diff --git a/dist/private/Menu.cjs b/dist/private/Menu.cjs
index d8abce09156486561c5e3806db37e65099631f08..381ed90dc9beda6bad2f0dc7a5fff6f15e3a2c12 100644
--- a/dist/private/Menu.cjs
+++ b/dist/private/Menu.cjs	
@@ -132,9 +132,7 @@ function $76f9e98b4261ab43$export$27d2ad3c5815583e(props) {
 }
 const $76f9e98b4261ab43$var$SubmenuTriggerContext = /*#__PURE__*/ (0, $fDx7A$react.createContext)(null);
 class $76f9e98b4261ab43$var$SubmenuTriggerNode extends (0, $fDx7A$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'submenutrigger';
-    }
+    static type = 'submenutrigger';
     filter(collection, newCollection, filterFn) {
         let triggerNode = collection.getItem(this.firstChildKey);
         if (triggerNode && filterFn(triggerNode.textValue, this)) {
diff --git a/dist/private/Menu.mjs b/dist/private/Menu.mjs
index c7f35b8d1c70110e2f1981286b248e689281250d..449cb97ed205b9a89f67b57ea265698e1cf768bc 100644
--- a/dist/private/Menu.mjs
+++ b/dist/private/Menu.mjs	
@@ -115,9 +115,7 @@ function $49319ee1285aa241$export$27d2ad3c5815583e(props) {
 }
 const $49319ee1285aa241$var$SubmenuTriggerContext = /*#__PURE__*/ (0, $7xe5e$createContext)(null);
 class $49319ee1285aa241$var$SubmenuTriggerNode extends (0, $7xe5e$CollectionNode) {
-    static{
-        this.type = 'submenutrigger';
-    }
+    static type = 'submenutrigger';
     filter(collection, newCollection, filterFn) {
         let triggerNode = collection.getItem(this.firstChildKey);
         if (triggerNode && filterFn(triggerNode.textValue, this)) {
diff --git a/dist/private/Separator.cjs b/dist/private/Separator.cjs
index 159486df308b41b07a4f830eda3b8e3d6d704eb4..887a62b442ca3024e51834789046b1149d2283ef 100644
--- a/dist/private/Separator.cjs
+++ b/dist/private/Separator.cjs	
@@ -36,9 +36,7 @@ $parcel$export(module.exports, "Separator", function () { return $5a1b0036f8cbf0
 
 const $5a1b0036f8cbf051$export$6615d83f6de245ce = /*#__PURE__*/ (0, $5zjXU$react.createContext)({});
 class $5a1b0036f8cbf051$export$7750289ca694c0b5 extends (0, $5zjXU$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'separator';
-    }
+    static type = 'separator';
     filter(collection, newCollection) {
         let prevItem = newCollection.getItem(this.prevKey);
         if (prevItem && prevItem.type !== 'separator') {
diff --git a/dist/private/Separator.mjs b/dist/private/Separator.mjs
index 5fc1a3324e341d9cab276ad0ab7bd4936b7a2035..ec876fabc7460708c6fe48ca7b5ebc1bf38d6b22 100644
--- a/dist/private/Separator.mjs
+++ b/dist/private/Separator.mjs	
@@ -25,9 +25,7 @@ import $adn6H$react, {createContext as $adn6H$createContext} from "react";
 
 const $e28ab3efe3e87743$export$6615d83f6de245ce = /*#__PURE__*/ (0, $adn6H$createContext)({});
 class $e28ab3efe3e87743$export$7750289ca694c0b5 extends (0, $adn6H$CollectionNode) {
-    static{
-        this.type = 'separator';
-    }
+    static type = 'separator';
     filter(collection, newCollection) {
         let prevItem = newCollection.getItem(this.prevKey);
         if (prevItem && prevItem.type !== 'separator') {
diff --git a/dist/private/Table.cjs b/dist/private/Table.cjs
index 1ad47dc3f5977aa3140de0ef02925b78eb79ecd7..f3b9727664d685355e6f7b00ef9f51ab2adda663 100644
--- a/dist/private/Table.cjs
+++ b/dist/private/Table.cjs	
@@ -572,9 +572,7 @@ function $af9b3e5459f1bf61$export$fddc468cd8cb4db9() {
     return (0, $ke19Y$react.useContext)($af9b3e5459f1bf61$var$TableOptionsContext);
 }
 class $af9b3e5459f1bf61$var$TableHeaderNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'tableheader';
-    }
+    static type = 'tableheader';
 }
 let $af9b3e5459f1bf61$var$THeadElementType = /*#__PURE__*/ (0, $ke19Y$react.forwardRef)(function THeadElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $ke19Y$react.useContext)((0, $f7b82bedbb70abac$exports.CollectionRendererContext));
@@ -676,9 +674,7 @@ function $af9b3e5459f1bf61$var$TableHeaderRow({ item: item }) {
     })));
 }
 class $af9b3e5459f1bf61$var$TableColumnNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'column';
-    }
+    static type = 'column';
 }
 let $af9b3e5459f1bf61$var$ColumnElementType = /*#__PURE__*/ (0, $ke19Y$react.forwardRef)(function ColumnElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $ke19Y$react.useContext)((0, $f7b82bedbb70abac$exports.CollectionRendererContext));
@@ -855,9 +851,7 @@ const $af9b3e5459f1bf61$export$ee689e97a7664bfd = /*#__PURE__*/ (0, $ke19Y$react
     }), document.body));
 });
 class $af9b3e5459f1bf61$var$TableBodyNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.FilterableNode) {
-    static{
-        this.type = 'tablebody';
-    }
+    static type = 'tablebody';
 }
 let $af9b3e5459f1bf61$var$TableBodyElementType = /*#__PURE__*/ (0, $ke19Y$react.forwardRef)(function TableBodyElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $ke19Y$react.useContext)((0, $f7b82bedbb70abac$exports.CollectionRendererContext));
@@ -931,9 +925,7 @@ const $af9b3e5459f1bf61$export$76ccd210b9029917 = /*#__PURE__*/ (0, $ke19Y$react
     }), emptyState);
 });
 class $af9b3e5459f1bf61$var$TableFooterNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.FilterableNode) {
-    static{
-        this.type = 'tablefooter';
-    }
+    static type = 'tablefooter';
 }
 let $af9b3e5459f1bf61$var$TableFooterElementType = /*#__PURE__*/ (0, $ke19Y$react.forwardRef)(function TableFooterElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $ke19Y$react.useContext)((0, $f7b82bedbb70abac$exports.CollectionRendererContext));
@@ -974,9 +966,7 @@ const $af9b3e5459f1bf61$export$1a75e308b53225a6 = /*#__PURE__*/ (0, $ke19Y$react
     isFocusVisibleWithinRow: false
 });
 class $af9b3e5459f1bf61$var$TableRowNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
     filter(collection, newCollection, filterFn) {
         let cells = collection.getChildren(this.key);
         for (let cell of cells)if (filterFn(cell.textValue, cell)) {
@@ -1176,9 +1166,7 @@ const $af9b3e5459f1bf61$export$b59bdbef9ce70de2 = /*#__PURE__*/ (0, $ke19Y$react
     }, props.children);
 });
 class $af9b3e5459f1bf61$var$TableCellNode extends (0, $ke19Y$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'cell';
-    }
+    static type = 'cell';
 }
 let $af9b3e5459f1bf61$var$TableCellElementType = /*#__PURE__*/ (0, $ke19Y$react.forwardRef)(function TableCellElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $ke19Y$react.useContext)((0, $f7b82bedbb70abac$exports.CollectionRendererContext));
diff --git a/dist/private/Table.mjs b/dist/private/Table.mjs
index d6d4de758bddd032842c608438a236cd40ff5e0a..b648224192cb1dc28d973b82f5efad63cce3b6bc 100644
--- a/dist/private/Table.mjs
+++ b/dist/private/Table.mjs	
@@ -553,9 +553,7 @@ function $76d00c5a4edb230a$export$fddc468cd8cb4db9() {
     return (0, $8WNJc$useContext)($76d00c5a4edb230a$var$TableOptionsContext);
 }
 class $76d00c5a4edb230a$var$TableHeaderNode extends (0, $8WNJc$CollectionNode) {
-    static{
-        this.type = 'tableheader';
-    }
+    static type = 'tableheader';
 }
 let $76d00c5a4edb230a$var$THeadElementType = /*#__PURE__*/ (0, $8WNJc$forwardRef)(function THeadElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $8WNJc$useContext)((0, $263ab7fc0f95ccdb$export$4feb769f8ddf26c5));
@@ -657,9 +655,7 @@ function $76d00c5a4edb230a$var$TableHeaderRow({ item: item }) {
     })));
 }
 class $76d00c5a4edb230a$var$TableColumnNode extends (0, $8WNJc$CollectionNode) {
-    static{
-        this.type = 'column';
-    }
+    static type = 'column';
 }
 let $76d00c5a4edb230a$var$ColumnElementType = /*#__PURE__*/ (0, $8WNJc$forwardRef)(function ColumnElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $8WNJc$useContext)((0, $263ab7fc0f95ccdb$export$4feb769f8ddf26c5));
@@ -836,9 +832,7 @@ const $76d00c5a4edb230a$export$ee689e97a7664bfd = /*#__PURE__*/ (0, $8WNJc$forwa
     }), document.body));
 });
 class $76d00c5a4edb230a$var$TableBodyNode extends (0, $8WNJc$FilterableNode) {
-    static{
-        this.type = 'tablebody';
-    }
+    static type = 'tablebody';
 }
 let $76d00c5a4edb230a$var$TableBodyElementType = /*#__PURE__*/ (0, $8WNJc$forwardRef)(function TableBodyElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $8WNJc$useContext)((0, $263ab7fc0f95ccdb$export$4feb769f8ddf26c5));
@@ -912,9 +906,7 @@ const $76d00c5a4edb230a$export$76ccd210b9029917 = /*#__PURE__*/ (0, $8WNJc$creat
     }), emptyState);
 });
 class $76d00c5a4edb230a$var$TableFooterNode extends (0, $8WNJc$FilterableNode) {
-    static{
-        this.type = 'tablefooter';
-    }
+    static type = 'tablefooter';
 }
 let $76d00c5a4edb230a$var$TableFooterElementType = /*#__PURE__*/ (0, $8WNJc$forwardRef)(function TableFooterElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $8WNJc$useContext)((0, $263ab7fc0f95ccdb$export$4feb769f8ddf26c5));
@@ -955,9 +947,7 @@ const $76d00c5a4edb230a$export$1a75e308b53225a6 = /*#__PURE__*/ (0, $8WNJc$creat
     isFocusVisibleWithinRow: false
 });
 class $76d00c5a4edb230a$var$TableRowNode extends (0, $8WNJc$CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
     filter(collection, newCollection, filterFn) {
         let cells = collection.getChildren(this.key);
         for (let cell of cells)if (filterFn(cell.textValue, cell)) {
@@ -1157,9 +1147,7 @@ const $76d00c5a4edb230a$export$b59bdbef9ce70de2 = /*#__PURE__*/ (0, $8WNJc$creat
     }, props.children);
 });
 class $76d00c5a4edb230a$var$TableCellNode extends (0, $8WNJc$CollectionNode) {
-    static{
-        this.type = 'cell';
-    }
+    static type = 'cell';
 }
 let $76d00c5a4edb230a$var$TableCellElementType = /*#__PURE__*/ (0, $8WNJc$forwardRef)(function TableCellElementType(props, ref) {
     let { isVirtualized: isVirtualized } = (0, $8WNJc$useContext)((0, $263ab7fc0f95ccdb$export$4feb769f8ddf26c5));
diff --git a/dist/private/Tabs.cjs b/dist/private/Tabs.cjs
index 0e7fbf925d24d1d5eaeb99a83d4ca5d55071d711..31b93900fdddcd75d700dcbbaa2d9290b3cbc557 100644
--- a/dist/private/Tabs.cjs
+++ b/dist/private/Tabs.cjs	
@@ -171,9 +171,7 @@ function $d6999420f0e5d757$var$TabListInner({ props: props, forwardedRef: ref })
     })));
 }
 class $d6999420f0e5d757$var$TabItemNode extends (0, $KH96U$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $d6999420f0e5d757$export$3e41faf802a29e71 = /*#__PURE__*/ (0, $KH96U$reactariaCollectionBuilder.createLeafComponent)($d6999420f0e5d757$var$TabItemNode, (props, forwardedRef, item)=>{
     let state = (0, $KH96U$react.useContext)($d6999420f0e5d757$export$364712098d2aa57c);
diff --git a/dist/private/Tabs.mjs b/dist/private/Tabs.mjs
index 184942b471af8ec62bb7fc4c73345b25f1663139..0e2c3ccb8875f9c0560aa6cb031f1f443ec7af38 100644
--- a/dist/private/Tabs.mjs
+++ b/dist/private/Tabs.mjs	
@@ -155,9 +155,7 @@ function $b4f18e3395fe64d7$var$TabListInner({ props: props, forwardedRef: ref })
     })));
 }
 class $b4f18e3395fe64d7$var$TabItemNode extends (0, $4KeVI$CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $b4f18e3395fe64d7$export$3e41faf802a29e71 = /*#__PURE__*/ (0, $4KeVI$createLeafComponent)($b4f18e3395fe64d7$var$TabItemNode, (props, forwardedRef, item)=>{
     let state = (0, $4KeVI$useContext)($b4f18e3395fe64d7$export$364712098d2aa57c);
diff --git a/dist/private/Tree.cjs b/dist/private/Tree.cjs
index 00a5a399f686cf28a601d611cd59049355e97df3..b9a92fb814ef69af90fb9f00e70967c65dd08b44 100644
--- a/dist/private/Tree.cjs
+++ b/dist/private/Tree.cjs	
@@ -400,9 +400,7 @@ function $8b1888e80597ea2d$var$TreeInner({ props: props, collection: collection,
     }))), emptyState)), dragPreview);
 }
 class $8b1888e80597ea2d$var$TreeContentNode extends (0, $av2Q0$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'content';
-    }
+    static type = 'content';
 }
 const $8b1888e80597ea2d$export$4b687e3f663d618c = /*#__PURE__*/ (0, $av2Q0$reactariaCollectionBuilder.createLeafComponent)($8b1888e80597ea2d$var$TreeContentNode, function TreeItemContent(props) {
     let values = (0, $av2Q0$react.useContext)($8b1888e80597ea2d$export$36b5dda0d9bc8f78);
@@ -416,9 +414,7 @@ const $8b1888e80597ea2d$export$4b687e3f663d618c = /*#__PURE__*/ (0, $av2Q0$react
 });
 const $8b1888e80597ea2d$export$36b5dda0d9bc8f78 = /*#__PURE__*/ (0, $av2Q0$react.createContext)(null);
 class $8b1888e80597ea2d$var$TreeItemNode extends (0, $av2Q0$reactariaprivatecollectionsBaseCollection.CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $8b1888e80597ea2d$export$53d36ab85dc89436 = /*#__PURE__*/ (0, $av2Q0$reactariaCollectionBuilder.createBranchComponent)($8b1888e80597ea2d$var$TreeItemNode, (props, ref, item)=>{
     let state = (0, $av2Q0$react.useContext)($8b1888e80597ea2d$export$8953bccafd7bce87);
diff --git a/dist/private/Tree.mjs b/dist/private/Tree.mjs
index 0f1b163a049bae380bea1db54200db3189f28314..4a7c1043125f8c6a8837eea604aa3a16134bd936 100644
--- a/dist/private/Tree.mjs
+++ b/dist/private/Tree.mjs	
@@ -383,9 +383,7 @@ function $89d060d8eb451127$var$TreeInner({ props: props, collection: collection,
     }))), emptyState)), dragPreview);
 }
 class $89d060d8eb451127$var$TreeContentNode extends (0, $YRtKP$CollectionNode) {
-    static{
-        this.type = 'content';
-    }
+    static type = 'content';
 }
 const $89d060d8eb451127$export$4b687e3f663d618c = /*#__PURE__*/ (0, $YRtKP$createLeafComponent)($89d060d8eb451127$var$TreeContentNode, function TreeItemContent(props) {
     let values = (0, $YRtKP$useContext)($89d060d8eb451127$export$36b5dda0d9bc8f78);
@@ -399,9 +397,7 @@ const $89d060d8eb451127$export$4b687e3f663d618c = /*#__PURE__*/ (0, $YRtKP$creat
 });
 const $89d060d8eb451127$export$36b5dda0d9bc8f78 = /*#__PURE__*/ (0, $YRtKP$createContext)(null);
 class $89d060d8eb451127$var$TreeItemNode extends (0, $YRtKP$CollectionNode) {
-    static{
-        this.type = 'item';
-    }
+    static type = 'item';
 }
 const $89d060d8eb451127$export$53d36ab85dc89436 = /*#__PURE__*/ (0, $YRtKP$createBranchComponent)($89d060d8eb451127$var$TreeItemNode, (props, ref, item)=>{
     let state = (0, $YRtKP$useContext)($89d060d8eb451127$export$8953bccafd7bce87);
